arm/Microsoft.ServiceBus/namespaces/deploy.bicep

```diff
@@ -141,8 +141,18 @@ var identity = identityType != 'None' ? {
   userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
 } : null
 
+
+var networkAcl = !empty(privateEndpoints) ? {
+  publicNetworkAccess: 'Enabled'
+  allowTrustedServices: true
+} : {
+  publicNetworkAccess: 'Disabled'
+  allowTrustedServices: false
```

Comment:
Why disabling MS trusted services by default if private endpoint is enabled?

Reply:
@eriqua - I have changed to default to true as per MS Docs information inside optional parameter in commit 6ca5b3e

Reply:
Secure defaults configured in latest commit and param override in place

Reply:
@eriqua : Looks good to me, thoughts?